client/app/email-domain/email-domain.service.js

```diff
@@ -737,7 +737,7 @@ angular.module("services").service(
             return this.OvhHttp.get("/email/domain/delegatedAccount", {
                 rootPath: "apiv6",
                 params: {
-                    accountName,
+                    accountName: accountName || undefined,
```

Comment:
Why do we need to send `undefined` to API ?

If `accountName` is not defined, we can just remove the parameter and don't it imo :)

Reply:
i changed the way to call API with facultatives params